cfgov/v1/signals.py

```diff
@@ -53,25 +53,26 @@ def update_all_revisions(instance, attr):
 
 
 def unshare_all_revisions(sender, **kwargs):
-    from v1.wagtail_hooks import flush_akamai
     update_all_revisions(kwargs['instance'], 'shared')
-    flush_akamai()
```

Comment:
I removed the flush call from unshare_all_revisions and unpublish_all_revisions since we will be removing this code soon anyway (yay!) and decided to hook it up directly to the page_published and page_unpublished signals.